### PR TITLE
USX UI Change 1

### PR DESCRIPTION
**Pull request by myothiha09**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @myothiha09 has signed the CLA. Thanks for the pull request!

**Comment by myothiha09** [comment]
PositiveNegativeDialogFragment has very trivial change (the USX version just change the theme)
The CollaboratorInitialView also has major changes in another PR. Please ignore the changes in this one.

**Comment by myothiha09** [comment]
CollabInitialView has some major changes on later PRs.

File: box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java [hunk 1]
```diff
@@ -60,8 +61,8 @@ protected void onCreate(Bundle savedInstanceState) {
         setTitle(R.string.box_sharesdk_sample_name);
 
         BoxConfig.IS_LOG_ENABLED = true;
-        BoxConfig.CLIENT_ID = "CLIENT_ID";
-        BoxConfig.CLIENT_SECRET = "CLIENT_SECRET";
+        BoxConfig.CLIENT_ID = "9itlbj9i8mh5qxpj5vr2pjy08t43n3yy";
+        BoxConfig.CLIENT_SECRET = "MVLrHgvscnPCpkNhiD2rrETuh2vNBftz";
```
Comment (nyjalusc) [inline box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java]: nice

File: box-share-sdk/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -107,6 +142,13 @@
         <item name="android:layout_gravity">left|start|center_vertical</item>
     </style>
 
+    <style name="BoxSwitchNew"  parent="@android:style/Widget.CompoundButton">
```
Comment (nyjalusc) [inline box-share-sdk/src/main/res/values/styles.xml]: Avoid using "*New" in any context.

**Pull request merged**